.github/workflows/ci.yaml

```diff
@@ -122,7 +122,7 @@ jobs:
          )
          || (
            github.event_name == 'push'
-           && github.ref == 'master'
+           && github.ref == 'refs/heads/master'
          )"
 
```

Comment:
This portion of the conditional seems unnecessary, push/merge events should be handled already by the preceeding lines, and exclude sbt-dotty tag pushes.

Reply:
Good point, we can merge this condition with the previous one.